docs/book/common_code/machine_controller.md

```diff
@@ -10,16 +10,21 @@ the host with a new one matching the updated spec. If a `Machine` object is
 deleted, the corresponding `Node` should have its external resources released by
 the provider-specific controller, and should be deleted as well.
 
+Machines can be associated with a Cluster using a custom label
+`cluster.k8s.io/clusterName`. Providers using the `Cluster` controller must
+provide the label which references the name of a cluster living in the same
+namespace.
```

Comment:
This reads a bit awkward to me as if the label is used by the `Cluster` controller.

What about:
```
Machines can optionally be associated with a Cluster using a custom label... If the label is set, then it must reference the name of a cluster residing in the same namespace.

Reply:
Reworded, thanks!